src/components/Content/ContentPanel.tsx

```diff
@@ -30,13 +33,7 @@ function normalizeRow(
 ) {
   const rowData = {
     ...row,
-    fieldContent:
-      row.fieldContent ||
-      (data[row.key] ? (
-        getFieldContent(row, data)
-      ) : (
-        <span style={{ opacity: 0.4, fontStyle: 'italic' }}>empty</span>
-      )),
+    fieldContent: row.fieldContent || getFieldContent(row, data),
```

Comment:
why not include the check for row.fieldContent in the getFieldContent method, then you can simplify this line.